otter/test/convergence/test_service.py

```diff
@@ -110,38 +440,54 @@ def setUp(self):
                        desired_lbs=self.desired_lbs,
                        links=freeze([{'href': 'link2', 'rel': 'self'}]))
         ]
+        gsgi = GetScalingGroupInfo(tenant_id='tenant-id',
+                                   group_id='group-id')
+        manifest = {  # Many details elided!
+            'state': self.state,
+            'launchConfiguration': self.lc,
+        }
+        gsgi_result = (self.group, manifest)
+        self.expected_intents = [(gsgi, gsgi_result)]
+
+    def _get_dispatcher(self, expected_intents=None):
```

Comment:
Hmm... this seems to have changed/moved since I made this comment.  Possibly I saw a cached/outdated version?  At any rate, this seems more specific now, although maybe a generalized version might be useful at some point in test/utils, my comment is less applicable to this PR now.  :)